polkadot-parachains/canvas-kusama/src/xcm_config.rs

```diff
@@ -170,7 +176,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Everything;
+	type XcmReserveTransferFilter = Nothing;
```

Comment:
Okay so confirm the intention of this change, this will mean that anybody will still be able to `Teleport` any asset in and out of Canvas, but will **not** be able to perform `RevervedTransfer`s, correct?

Reply:
They should probably both be set to `Nothing` at genesis.

Reply:
If we set both to `Nothing` then users won't be able to get `ROC` tokens on Canvas to pay for fees. So for `Teleport` maybe it should be `EverythingBut<IsParent>` instead?

Reply:
Oh I didn't know this was what is used on Rococo. Then yes.